### DISCUSSION TITLE
Is it any way to run it without Docker?

### DISCUSSION THREAD

**Discussion by ke1ne** [open-webui/open-webui#819] [Q&A] [answered]
Well, just like asked. Thanks!

**Comment by justinh-rahb** [comment, marked as answer]
At your own risk: [README.md](https://github.com/open-webui/open-webui?tab=readme-ov-file#how-to-install-without-docker)

**Comment by ke1ne** [comment]
> At your own risk: [README.md](https://github.com/open-webui/open-webui?tab=readme-ov-file#how-to-install-without-docker)

OMG, I did lost that part. Thank you!
